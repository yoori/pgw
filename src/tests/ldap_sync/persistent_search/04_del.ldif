#
# ARGV: -x -H ${PERSISTENT_SEARCH_TEST_SERVER} -D "cn=Directory Manager" -w "secret123"
# OUT:peopleDeletebill
#
dn: uid=bill,ou=people,dc=example,dc=com
changetype: delete

