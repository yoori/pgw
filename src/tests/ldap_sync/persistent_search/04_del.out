Delete ou=people,dc=example,dc=com uid=bill,ou=people,dc=example,dc=com  bill
